tracer/src/Datadog.Trace/Iast/Aspects/System.Runtime/DefaultInterpolatedStringHandlerAspect.cs

```diff
@@ -0,0 +1,155 @@
+// <copyright file="DefaultInterpolatedStringHandlerAspect.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NET6_0_OR_GREATER
+
+using System;
+using System.Reflection.Emit;
+using System.Runtime.CompilerServices;
+using Datadog.Trace.Iast.Dataflow;
+using Datadog.Trace.Iast.Propagation;
+using InlineIL;
+using static InlineIL.IL.Emit;
+
+#nullable enable
+
+namespace Datadog.Trace.Iast.Aspects.System.Runtime;
+
+/// <summary> DefaultInterpolatedString class aspect </summary>
+[AspectClass("System.Runtime")]
+[global::System.ComponentModel.Browsable(false)]
+[global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
+public class DefaultInterpolatedStringHandlerAspect
```

Comment:
We should probably cover also the ToString() method?

Reply:
Unfortunately, the ToString() method is not called on the struct but as a System.Object. 😢 
![image](https://github.com/user-attachments/assets/7a6b652a-3086-426a-bac0-ec050d2dc80a)

Reply:
Apart from that, these classes are never used explicitly, the calls are generated by the compiler, and it never uses `ToString()`